health/REFERENCE.md

```diff
@@ -390,6 +390,10 @@ but, we also have for the workstations the following `netdata.conf`
     room = workstation
 ```
 
+Before we proceed, it is necessary to explain something about possible values for labels. The labels do not have restrictions
+ for their values, but their name only accepts ASCII characters, but it does not accept all the ASCII values, the 
+ label's name only can contain alphabet letters and numbers.
+
```

Comment:
i think it was alnum + _ + - ?

Reply:
Thank you, I rewrite the documentation and I moved it for the correct place.